pyproject.toml

```diff
@@ -87,6 +88,11 @@ ignore_whitespace = true
 module = ["ape.*", "twisted.*"]
 ignore_missing_imports = true
 
+[[tool.mypy.overrides]]
+module = ["beamer.tests.*"]
+disallow_untyped_defs = false
+disable_error_code = "attr-defined,name-defined"
+
```

Comment:
Why is this hunk here? I think we had this issue before, where _something_ (poetry?) would accidentally add this hunk.